c++/src/io/OutputStream.cc

```diff
@@ -37,7 +37,7 @@ namespace orc {
                                     : outputStream(outStream),
                                       blockSize(blockSize_),
                                       metrics(metrics_) {
-    dataBuffer.reset(new DataBuffer<char>(pool));
+    dataBuffer.reset(new BlockBuffer(pool, blockSize));
```

Comment:
Does the **blockSize_** here have the same meaning of BlockBuffer?

Reply:
Before this patch, Data buffer will increase memory in blockSize_ during the next() function of class BufferedOutputStream. So the blockSize_ is reasonable as a construction parameter for class BlockBuffer.

Reply:
OK, that makes sense.